examples/tutorial/2drun.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+build/run_mcmc \
+  --algo-params-file resources/tutorial/algo.asciipb \
+  --hier-type NNW --hier-args resources/asciipb/nnw_ngiw_prior.asciipb \
```

Comment:
This file seems to be missing (also we don't use the asciipb folder anymore). Can you please move it to resources/tutorial and upload it?